math/binomial.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @brief Program to calculate binomial coefficients.
```

Comment:
Provide a Wikipedia link (if available/possible) in Markdown format.
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂